### DISCUSSION TITLE
Error after upgrade to 3.5.5

### DISCUSSION THREAD

**Discussion by arleinalesso** [netbox-community/netbox#13189] [Help Wanted!]
### NetBox version

v3.5.5 or v3.5.6

### Python version

3.8

### Steps to Reproduce

After upgrade to version 3.5.5, netbox not run corretly.

### Expected Behavior

Simply run

### Observed Behavior

This error

![image](https://github.com/netbox-community/netbox/assets/2326549/a259c168-6554-48fa-9cac-1066f69fb769)

**Comment by candlerb** [comment]
The string "ImageDraw" doesn't occur anywhere in the Netbox source code, so do you have any plugins installed? If so, disable them and try again.

Otherwise, what version of Netbox are you upgrading *from*?

**Comment by joaolucasmacedo** [comment]
Hi @arleinalesso I had the same issue.
Looks like it is something related to "Pillow" package v10. 
You need to install `Pillow==9.5.0`.

**Reply by candlerb** [reply]
Can you explain in more detail what the problem you found was, and why that fix was required?

Netbox explicitly pins `Pillow==10.0.0` in its requirements.txt, so by changing this, you are installing it in a way contrary to how it is installed and tested by the developers.

EDIT: it is true that ImageDraw.textsize has been removed, according to the [release notes](https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html#font-size-and-offset-methods). The question is, what code was *calling* ImageDraw.textsize?

If you can provide a backtrace this will make it clear.

If it's a plugin for example, then the correct solution is to update (or disable) the plugin, not to break Netbox.

**Reply by joaolucasmacedo** [reply]
Now that I've read #13115 I see that it is related to QRcode plugin.
In my case, i'm using Netbox v3.5.6 and Python 3.10. I just installed the package with pip.

**Comment by abhi1693** [comment]
The issue is due to mostly https://github.com/FloMeyer/netbox-qr and can be seen in https://github.com/netbox-community/netbox/issues/13115

**Comment by arleinalesso** [comment]
Thanks abhi1693.
Netbox up and run!!!!!
